src/ui-shell/sidenav/sidenav-item.component.ts

```diff
@@ -58,19 +63,21 @@ export class SideNavItem {
 	 */
 	@Input() route: any[];
 
+	@Input() content: string;
```

Comment:
I tried passing in the content through `ng-content` by taking the `TemplateRef` of the header items and header menus but that caused a lot of issues. So we opted for a model approach. I thought that it would be easier for the user to pass in the same model to both the sidenav and header but let me know if there is a better way